config/peerpods/podvm/podvm-handling.md

```diff
@@ -35,7 +35,9 @@ image creation process may be re-triggered.
 
 ## Brief description of the K8s job manifests
 
-`osc-podvm-image-creation.yaml`: This job manifest is used to create the pod VM
```

Comment:
hmm I think it's correct... it mean to describe the *manifest* (yaml) files.

Reply:
You are right. Dropping this commit.